frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/AggregationItem/AggregationItem.tsx

```diff
@@ -20,27 +23,19 @@ export function AggregationItem({
   aggregation,
   aggregationIndex,
   onQueryChange,
+  displayName,
+  handleRemove,
+  operators,
 }: AggregationItemProps) {
-  const [isOpened, setIsOpened] = useState(false);
```

Comment:
this data is moved into `useQueryAggregations` - we can use this data in the SDK `SummarizePicker` component